docs/src/pages/guides/composition/composition.md

```diff
@@ -9,11 +9,11 @@ we need a way to know the nature of the child elements a component receives.
 To solve this problem we tag some of our components when needed
 with a `muiName` static property.
 
-However, users like to wrap components in order to enhance them.
-That can conflict with our `muiName` solution. If you wrap a component verify if
+However, you may need to wrap a component in order to enhance it, 
+which can conflict with the muiName solution. If you wrap a component verify if
```

Comment:
`muiName`?

Reply:
I would keep the code formatting since it's a variable name of sorts.

Reply:
Just pointing out that the backticks got lost somewhere along the way.